### ISSUE TITLE
Error with @launchpad_openjdk_gpg

### ISSUE THREAD

**Issue by c-parsons** [bazelbuild/rules_docker#1037]
BazelCI against rules_docker is [failing at HEAD](https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/1110#e4c747cd-777c-49ce-b8b2-7a0529be1011)

It looks like an error with the dependency on `@launchpad_openjdk_gpg`. The sha seems to have changed.

A naive fix should be to simply update the sha256, but I'd like to leave it to rules_docker owners to understand why the contents of the remote file may have changed...

```
(02:12:47) ERROR: /var/lib/buildkite-agent/builds/bk-docker-879v/bazel-downstream-projects/rules_docker/tests/docker/package_managers/BUILD:103:1: //tests/docker/package_managers:gpg_image depends on @launchpad_openjdk_gpg//file:file in repository @launchpad_openjdk_gpg which failed to fetch. no such package '@launchpad_openjdk_gpg//file':

java.io.IOException: Error downloading [http://keyserver.ubuntu.com/pks/lookup?op=get&fingerprint=on&search=0xEB9B1D8886F44E2A] to /var/lib/buildkite-agent/.cache/bazel/_bazel_buildkite-agent/e3859f22398e3a8fd08acadeaa226d91/external/launchpad_openjdk_gpg/file/downloaded: 

Checksum was 429b667af38f71f727de92aa662fb535c587b870a8ed0dcfbe7d2b8971b7e616 but wanted 54b6274820df34a936ccc6f5cb725a9b7bb46075db7faf0ef7e2d86452fa09fd
```

**Comment by alex1545** [comment]
Hi @c-parsons and thank you for opening this issue.
We noticed this error earlier today and it's already addressed in #1036

**Comment by c-parsons** [comment]
Awesome. Thanks!
